pom.xml

```diff
@@ -474,12 +428,6 @@
         </exclusions>
       </dependency>
       <dependency>
-        <groupId>org.javassist</groupId>
```

Comment:
Who did you determine that `javassist` is not needed? I vaguely recall seeing this in relation to the Sonar reporting... but could be wrong.